roofit/batchcompute/inc/rbc.h

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
This`rbc.h` header doesn't exist anymore now: it was merged with `RooBatchCompute.h`. Now the only new header in `roobatchcmpute` introduced by this PR is `RooBatchCompute/Batches.h`, and indeed I noticed that it can go into `src`. I'll do that!